packages/nimble-tokens/data/fonts.json

```diff
@@ -858,6 +858,8 @@
       "type": "font",
       "id": "3850253f-2169-4b7a-a7b2-3d7dc78c2997",
       "name": "Error_ColorUI",
+      "last_updated": "2022-10-20T19:56:47.845Z",
```

Comment:
I debated on reverting the unnecessary changes made to this file (e.g. these lines, the updated ids below), but decided not to change what the XD plugin produced. If anyone thinks they should be reverted, let me know, and I can do that.

Reply:
Yea, just check-in whatever the XD plugin spits out. If manual changes are needed we would need to document it so that it can be done reliably. I don't see a benefit to adding the work to manually prevent these changes (the ideal is not checking in generated code but XD plugin doesn't give a programmatic API 😞).